packages/dash-spv/lib/spvchain.js

```diff
@@ -1,17 +1,37 @@
-const BlockStore = require('./blockstore');
 const config = require('../config/config');
 const Consensus = require('./consensus');
 const utils = require('./utils');
+const SPVError = require('./errors/SPVError');
 
 const SpvChain = class {
-  constructor(chainType, confirms = 100, startBlock) {
-    this.root = null;
-    this.allBranches = [];
-    this.orphanBlocks = [];
-    this.orphanChunks = [];
+  constructor(chainType, confirms = 100, startBlock, startBlockHeight) {
```

Comment:
Default param in the middle.

Reply:
Let's leave it for now, will be refactored into `options` in a dedicated ticket